skytap/vm.go

```diff
@@ -0,0 +1,336 @@
+package skytap
+
+import "context"
+
+// Default URL paths
+const (
+	vmsLegacyBasePath = "/configurations/"
```

Comment:
this can be more generic but happy to keep it as its a pattern use in other resources.

Reply:
ok